### ISSUE TITLE
Projected coordinate system

### ISSUE THREAD

**Issue by greatathrun** [uber/h3#121]
For those functions related to latitude and longitude, is the projected coordinate system used as WGS84 or something?

**Comment by sahrk** [comment]
It is a sphere with the WGS84 authalic radius (i.e., a sphere with the same surface area as the WGS84  spheroid).

**Comment by nrabinowitz** [comment]
Closing as answered - thanks @sahrk!

**Comment by nrabinowitz** [comment]
On second thought - reopening this until we include the answer in the core documentation.

**Comment by zachasme** [comment]
So should we assume coordinate system WGS84 or something like EPSG:3857 (when assigning an SRID to coordinates in PostGIS for example)?

**Comment by nagytech** [comment]
I think it would be valuable to at least specify WGS84 as a coordinate system.  

In terms of SRIDs, there are always trade-offs between the different spatial references.  Specifying EPSG:3857 in the documentation would be a good starting point and may help solve [these types of questions](https://gis.stackexchange.com/questions/342845/distorted-h3-hexagons-even-though-projection-is-matching/353042).

**Comment by Komzpa** [comment]
WGS84 2D lon-lat is EPSG:4326, not EPSG:3857.
You can try stealing def from GeoJSON spec:
https://tools.ietf.org/html/rfc7946
```
4.  Coordinate Reference System

   The coordinate reference system for all GeoJSON coordinates is a
   geographic coordinate reference system, using the World Geodetic
   System 1984 (WGS 84) [WGS84] datum, with longitude and latitude units
   of decimal degrees.  This is equivalent to the coordinate reference
   system identified by the Open Geospatial Consortium (OGC) URN
   urn:ogc:def:crs:OGC::CRS84.  An OPTIONAL third-position element SHALL
   be the height in meters above or below the WGS 84 reference
   ellipsoid.  In the absence of elevation values, applications
   sensitive to height or depth SHOULD interpret positions as being at
   local ground or sea level.
```

**Comment by ajfriend** [comment]
Closed with #455
